src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2295,13 +2295,34 @@ private List<DependentAssembly> GetAssemblyRemappingsFromAppConfig()
                     _cache.SetInstalledAssemblyInformation(installedAssemblyTableInfo);
 
                     // Cache delegates.
-                    getAssemblyName = _cache.CacheDelegate(getAssemblyName);
```

Comment:
nit:
Any reason to not remove _cache.SetGetLastWriteTime above?

Reply:
Nice catch! Done.